src/client/refresh/vk/header/vk_local.h

```diff
@@ -227,7 +227,9 @@ qboolean	R_EndWorldRenderReady( void );
 struct image_s *R_RegisterSkin (char *name);
 
 void LoadPCX (char *filename, byte **pic, byte **palette, int *width, int *height);
-image_t *Vk_LoadPic (char *name, byte *pic, int width, int height, imagetype_t type, int bits, qvksampler_t *samplerType);
+image_t *Vk_LoadPic(char *name, byte *pic, int width, int realwidth,
```

Comment:
More mixed functional and formatting changes.

Reply:
thank you, all codestyle changes moved to 695a9e9